rollup.intgConfig.js

```diff
@@ -108,7 +108,23 @@ export default {
     babel({
       babelHelpers: "bundled",
       exclude: ["node_modules/@babel/**", "node_modules/core-js/**"],
-      presets: [["@babel/env"]],
+      presets: [
+        [
+          "@babel/env",
+          {
+            corejs: "3.6",
+            useBuiltIns: "entry",
+            targets: {
+              edge: "15",
+              firefox: "40",
+              ie: "10",
+              chrome: "37",
+              safari: "7",
+              opera: "23",
+            },
+          },
+        ],
+      ],
```

Comment:
This doesn’t have any benefit unless you are importing core-js to your project. [Ref](https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry)

Reply:
True. It's a preemptive configuration.

Reply:
Let’s remove it then.

Reply:
This is added to serve as a source of truth for browser support. We might have to update the values after final testing.